diesel_tests/tests/connection.rs

```diff
@@ -2,22 +2,21 @@ use crate::schema::*;
 use diesel::connection::BoxableConnection;
 use diesel::*;
 
-table! {
```

Comment:
Moving around & changing code that's not necessary as part of the new or updated test code should be avoided, i.e. usually the minimal set of changes to resolve an issue is preferable.
If this is indeed updating some test code to follow newer and better patterns that are already in use elsewhere, that's of course acceptable and probably desirable. I don't know the specifics of this case, but I wanted to leave this general note so you could act on it based on your best judgment.

Reply:
@seeekr the PR has been updated to keep only ticket related changes. Thanks for your feedback.